tests/py3-ignores.txt

```diff
@@ -27,8 +25,6 @@ tests/test_engine.py
 tests/test_http_cookies.py
 tests/test_logformatter.py
 tests/test_mail.py
-tests/test_pipeline_files.py
-tests/test_pipeline_images.py
```

Comment:
> (where would that encoding be defined?)

That's exactly the problem - header encoding is not defined anywhere. In Cookies RFC it is said that client should try decode them using utf-8; many clients decode headers using latin1 (see e.g. PEP3333). Anything non-ascii is a gray area where server and client must agree on encoding, and standards don't help.